Gopkg.toml

```diff
@@ -47,3 +47,11 @@
 [[constraint]]
   name = "github.com/fatih/color"
   version = "1.5.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
I don't think this is versioned.

Reply:
Oh, weird, they seem to version the actual code _within_ the repo: i.e., every package has a version number in its name, such as `spanner/v1` and `calendar/v3`. I guess we'll end up always pulling down latest and then have to `import` the right versions...